21/debian/bookworm-slim/hotspot/Dockerfile

```
@@ -1,23 +1,6 @@
-ARG BOOKWORM_TAG=20230904
-FROM debian:bookworm-"${BOOKWORM_TAG}"-slim as jre-build
-ARG JAVA_VERSION=21+35
-ARG TARGETPLATFORM
-
-SHELL ["/bin/bash", "-o", "pipefail", "-c"]
-
-RUN set -x; apt-get update \
-  && apt-get install --no-install-recommends -y \
-    ca-certificates \
-    jq \
-    wget \
-  && BUILD_NUMBER=$(echo $JAVA_VERSION | cut -d'+' -f2) \
-  && JAVA_VERSION_ENCODED=$(echo "$JAVA_VERSION" | jq "@uri" -jRr) \
-  && CONVERTED_ARCH=$(arch | sed -e 's/x86_64/x64/' -e 's/armv7l/arm/') \
-  && wget --quiet https://github.com/adoptium/temurin21-binaries/releases/download/jdk-"${JAVA_VERSION_ENCODED}"-ea-beta/OpenJDK21U-jdk_"${CONVERTED_ARCH}"_linux_hotspot_ea_21-0-"${BUILD_NUMBER}".tar.gz -O /tmp/jdk.tar.gz \
-  && tar -xzf /tmp/jdk.tar.gz -C /opt/ \
-  && rm -f /tmp/jdk.tar.gz
-
-ENV PATH=/opt/jdk-${JAVA_VERSION}/bin:$PATH
+ARG BOOKWORM_TAG=20230919
```

```suggestion
ARG BOOKWORM_TAG=20230904
```

Worth keeping the same base and let the automated system updating this instead of "hiding" it in this PR.